src/shared/sol-conffile.c

```diff
@@ -37,9 +37,16 @@
 #include "sol-memmap-storage.h"
 #endif
 
+struct sol_alias_ctx {
+    char *node;
```

Comment:
store as `struct sol_str_slice` then you can get a more efficient `_eq()` that first checks the length and avoid strcmp()